packages/downloads/test/poll-download-metadata.test.ts

```diff
@@ -11,7 +11,7 @@ describe("pollDownloadMetadata", () => {
       spyOn(mockEventEmitter, "emit");
 
       spyOn(hubPoller, "hubPollDownloadMetadata").and.returnValue(
-        new Promise((resolve, reject) => {
+        new Promise((resolve, _reject) => {
```

Comment:
Some linters will fail if a supplied parameter is unused, but this can sometimes make it awkward for some callbacks like (err, data) in node and (resolve, reject) in promises. The underscore prefix denotes that you acknowledge that a param is being supplied but you will not use it. This is also done in a few places in the indexer!